crates/codegen/runtime/cargo/src/runtime/bindings/mod.rs

```diff
@@ -1,25 +1,42 @@
 #[path = "generated/binding_rules.rs"]
 mod binding_rules;
 
+#[path = "generated/built_ins.rs"]
+mod built_ins;
+
 use std::sync::Arc;
 
 use metaslang_bindings::{self, PathResolver};
-use semver::Version;
 
 use crate::cst::KindTypes;
+use crate::parser::Parser;
 
 pub type Bindings = metaslang_bindings::Bindings<KindTypes>;
 pub type Definition<'a> = metaslang_bindings::Definition<'a, KindTypes>;
 pub type Reference<'a> = metaslang_bindings::Reference<'a, KindTypes>;
 
 pub fn create_with_resolver(
-    version: Version,
+    parser: &Parser,
```

Comment:
not sure I understand the change here, given that `Bindings::add_built_ins()` (and `Bindings::add_file()`) exist in the public API. The `Bindings` owner is responsible for loading/parsing the built ins file, and passing its cursor, similar to everything else.
Thoughts?

Reply:
Yeah, this was before we decided to move loading the built-ins responsibility away from the Bindings API. I was going to make that change in another PR, but may as well do it here, along with the other issues we've discussed so far.